R/mobr_boxplots.R

```diff
@@ -98,20 +105,25 @@ repl_boot_samples <- function(comm_dat, n_rare, n_perm = 100)
    return(repl_biodiv)
 }
 
-# get confidence interval from bootstrap samples
-get_CI <- function(x, level = 0.95)
+# get mean and confidence interval from bootstrap samples
```

Comment:
Usually I think we opt for the median CI rather than the mean CI but this may not make a big difference if the CI's are symmetrical